.github/workflows/TestFunctional.yml

```diff
@@ -84,11 +92,19 @@ jobs:
 
       - uses: ./drainpipe/scaffold/github/actions/common/set-env
 
+      - name: Turn on DDEV debug logging
+        run: |
+          echo "DDEV_DEBUG=true" >> $GITHUB_ENV
+
       - name: Install DDEV
         uses: ./drainpipe/scaffold/github/actions/common/ddev
         with:
           git-name: Drainpipe Bot
-          git-email: no-reply@example.com#
```

Comment:
@justafish is this a typo we should fix?

Reply:
yes :smile: https://github.com/Lullabot/drainpipe/pull/527